File: asgmt_001/sum.jl
tfunction sum_of_a_million()
  sum = 0
  for i in 1:1000000
    sum += i
  end
  return sum
end

total = 0
for i in 1:10
  global total
  start = time_ns()
  sum_mill = sum_of_a_million()
  total += (time_ns() - start)
end
println("Average Time in Julia: ",total/10000000000)

File: asgmt_001/sum.py
import time

def sum_of_a_million():
  start = time.time()
  result = 0
  for i in range(1,1000000):
    result += i
  end = time.time()
  return end-start

sum_times = 0
for i in range(1,10):
  result = sum_of_a_million()
  sum_times += result
print("\nAverage Time in Python:", sum_times/10)
